spec/oncocore.txt

```diff
@@ -134,8 +134,8 @@ EntryElement:   Tumor
 Based on:		BodyStructure
 Concept:        SCT#395557000 "Tumor finding (finding)"
 Description:    "The presence of an abnormal mass of tissue (neoplasm) that results when cells divide more than they should or do not die when they should. Tumors may be benign (not cancer), or malignant (cancer). (source: NCI Dictionary)."
-                Code from HistologyMorphologyBehaviorVS
-0..1			RelatedCancerCondition
+                Code from HistologyMorphologyBehaviorVS if covered
+0..1			ref(PrimaryCancerCondition)
```

Comment:
A `Tumor` can also be associated with a `SecondaryCancerCondition`, so we should allow for that reference here as well.